src/Umbraco.Forms.Integrations.Crm.Hubspot/Api/Management/Controllers/Forms/GetAuthenticationUrlController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Forms.Integrations.Crm.Hubspot.Services;
+
+namespace Umbraco.Forms.Integrations.Crm.Hubspot.Api.Management.Controllers.Forms
+{
+    public class GetAuthenticationUrlController : ContactControllerBase
+    {
+        public GetAuthenticationUrlController(IContactService contactService) : base(contactService)
+        {
+        }
+
+        [HttpGet("auth/url")]
```

Comment:
This got me thinking, should we also do this for `AuthorizeController` and `DeauthorizeController` as well?
- `/auth/authorize`
- `/auth/deauthorize`
- `/auth/url`
- `/auth/configured`